Plutarch/Api/V1/AssocMap.hs

```diff
@@ -714,6 +717,7 @@ zipMergeInsertCommutative (MergeHandlerCommutative bothPresent onePresent) = unT
           PassOne -> pcons # x # xs'
           HandleOne handler ->
             List.pmap
+              {- HLINT ignore "Redundant bracket" -}
```

Comment:
This is generally pretty safe, was there a problem with hls applied solution? Fyi, I usually visit this file and HLS provides an action that refactors the thing. I never had a semantic issue with it.